### ISSUE TITLE
About hscnn-d

### ISSUE THREAD

**Issue by Lassi-Ki** [ngchc/HSCNN-Plus#9]
Can pytorch do the code of hscnn-d?
why choose tensorflow, does it have any advantage?
